### ISSUE TITLE
Duplicate editor in inproceedings (as in #86)

### ISSUE THREAD

**Issue by bjacquemin** [plk/biblatex-apa#110]
When citing an @inproceedings in biblatex-apa, I am getting the volume editor's(s') name(s) printed twice in the reference list. MWE:
`\documentclass{article}

\usepackage{polyglossia}
\setdefaultlanguage[variant=american]{english}
\usepackage{csquotes}
\usepackage[%
  backend=biber,
  style=apa,
 ]{biblatex}
\usepackage{filecontents}
\begin{filecontents}{minwex.bib}
@INPROCEEDINGS{salam,
  author       = {Salam, Abdus},
  editor       = {Svartholm, Nils},
  title        = {Weak and Electromagnetic Interactions},
  date         = {1968},
  booktitle    = {Elementary particle theory},
  eventdate    = {1968-05-19/1968-05-25},
  venue        = {Aspen{\"a}sgarden, Lerum},
  publisher    = {Almquist \& Wiksell},
  location     = {Stockholm},
  pages        = {367-377},
}
\end{filecontents}
\addbibresource{minwex.bib}

\begin{document}
\nocite{*}
\printbibliography%[title=Bibliographie]
\end{document}`
The produced reference contains the editor twice:
![test-apa-biblatex](https://user-images.githubusercontent.com/64803138/81000089-0884d780-8e46-11ea-97ba-97371420c29e.png)
Is there a way to solve the problem?
Thank you for all this APA work.

**Comment by moewew** [comment]
The duplicate `editor`s issue is also discussed in #106. 

```latex
\documentclass{article}

\usepackage[backend=biber, style=apa,]{biblatex}

\addbibresource{biblatex-examples.bib}

\begin{document}
\nocite{salam,moraux,pines,westfahl:space}
\printbibliography
\end{document}
```
gives
> Moraux, P. (1979). Le De Anima dans la tradition gr` ecque: Quelques aspects de l’interpretation du trait´ e, de Theophraste à Themistius In Aristotle on Mind and the Senses [Proceedings of the Seventh Symposium Aristotelicum] (G. E. R. Lloyd & G. E. L. Owen, Eds.). Cambridge, Cambridge University Press. 281–324

According to https://apastyle.apa.org/style-grammar-guidelines/references/examples/conference-proceeding-references `@inproceedings` should look pretty much the same as `@incollection` (https://apastyle.apa.org/style-grammar-guidelines/references/examples/edited-book-chapter-references), so there are two issues

1. The `editor`s should be in front of the title
2. the `pages` field should be in parentheses after the title.

**Comment by plk** [comment]
Should be fixed in 9.12, just released. I hadn't seen these web updates - they basically confirm that `proceedings/inproceedings` are just aliases to `book/inbook/article` which is how they are now implemented, with suitable mappings.

**Comment by bjacquemin** [comment]
Just tested 9.12, and it works fine for me.
Thank you for all this work.
